### PR TITLE
Various changes and improvements

### PR DESCRIPTION
**Pull request by luzzif**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -7,6 +7,10 @@
             name="description"
             content="Seamless, cheap and fast L2 token swapping built on Ethereum. Join the fast lane."
         />
+        <link
+            href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500;600;700&display=swap"
+            rel="stylesheet"
+        />
```
Comment (xiaowheat) [inline public/index.html:13]: @mfinestone @luzzif 

Using fonts.googleapis.com doesn't work in China. It's the reason that we would like to not load the fonts from a CDN

**Review by xiaowheat — APPROVED** [review]

**Review by xiaowheat — CHANGES_REQUESTED** [review]
Not using googleapis.com to load font.

File: public/index.html (hunk 1 above)
Comment (xiaowheat) [reply public/index.html:13]: From our discussion, I will approve the PR. Users in China won't get the fonts.

**Review by xiaowheat — APPROVED** [review]

File: public/index.html (hunk 1 above)
Comment (luzzif) [reply public/index.html:13]: Oh, sorry, didn't know that! I'm going to submit a second PR with locally hosted fonts.

**Pull request merged**